src/test/apps.test.js

```diff
@@ -555,3 +559,62 @@ describe("App validator", function () {
     expect(errors[2].attribute).to.equal("disk");
   });
 });
+
+describe("App Page component", function () {
+
+  beforeEach(function () {
+    var app = _.extend(appScheme, {
```

Comment:
Do we really need all this data for the tasks? 
I like only to define the mandatory properties.